optimism/src/main.rs

```diff
@@ -40,12 +40,113 @@ pub fn main() -> ExitCode {
 
     env_logger::Builder::from_env(env_logger::Env::default().default_filter_or("info")).init();
 
-    let mut env = witness::Env::<ark_bn254::Fq>::create(cannon::PAGE_SIZE as usize, state, po);
+    let domain_size = 1 << 15;
+
+    let domain =
+        kimchi::circuits::domains::EvaluationDomains::<ark_bn254::Fr>::create(domain_size).unwrap();
+
+    let srs = {
```

Comment:
Follow-up: use a public SRS

Reply:
Opened https://github.com/o1-labs/proof-systems/issues/1648 and linked in the code, see https://github.com/o1-labs/proof-systems/pull/1626/commits/0e44be9d8da27a6effd569833feffbb3f5299dd0

Reply:
> use a public SRS

do you mean available SRS's which are known to having been created in a ceremony instead of generated in the code?